glue/qt/widget_properties.py

```diff
@@ -237,9 +247,15 @@ def connect_current_combo(client, prop, widget):
     """
 
     def _push_combo(value):
-        idx = widget.findData(value)
-        if idx == -1:
-            return
+        # NOTE: we can't use findData here because if the data is not a 
+        # string, PySide will crash
```

Comment:
I've been bitten by `findData` like this too. We could envision fixing this method in our qt import/wrapper utility, so that findData behaves as we expect.